docs/src/components/CodeSnippet.tsx

```diff
@@ -6,24 +6,24 @@ import Editor, { EDITOR_BACKGROUND_COLOR } from './Editor'
 export interface CodeSnippetProps {
   fitted?: boolean
   label?: string
-  mode?: 'jsx' | 'html' | 'sh'
-  value: string | string[]
+  mode?: 'json' | 'jsx' | 'html' | 'sh'
+  value: string | string[] | Object
   style?: React.CSSProperties
 }
 
-const joinToString = (stringOrArray: string | string[]) => {
-  return typeof stringOrArray === 'string' ? stringOrArray : stringOrArray.join('\n')
-}
+const joinToString = (stringOrArray: string | string[]) =>
+  Array.isArray(stringOrArray) ? stringOrArray.join('\n') : stringOrArray
```

Comment:
could you, please, suggest why this is better?

Reply:
I rewrote this place, we accept a plain object there for JSON. Nice catch 👍